morse_code.rb

```diff
@@ -0,0 +1,25 @@
+def decode_char(char)
+  morse_char = {
+    A: '.-', B: '-...', C: '-.-.', D: '-..', E: '.',
+    F: '..-.', G: '--.', H: '....', I: '..', J: '.---',
+    K: '-.-', L: '.-..', M: '--', N: '-.', O: '---',
+    P: '.--.', Q: '--.-', R: '.-.', S: '...', T: '-',
+    U: '..-', V: '...-', W: '.--', X: '-..-', Y: '-.--', Z: '--..'
+  }
+
+  morse_char.key(char)
+end
+
+def decode_word(word)
+  new_word = word.split
+  word_result = new_word.map { |char| decode_char(char) }
+  word_result.join
+end
+
+def decode_message(message)
+  new_message = message.split('  ')
```

Comment:
- [ ] There is an extra space from the decoded message. To fix this, kindly add one more space to the `message.split('  ')` on line `20` so that the split method has `3` spaces inside it. Once you do this, the decoded sentence will have one space between each sentence.

Your Implementation | Afte I added extra space
:---:|:---:
![](https://i.imgur.com/UuIbJtx.png) | ![](https://i.imgur.com/DvtIkvC.png)